src/profile.c

```diff
@@ -487,13 +483,61 @@ static inline void printOutput(UInt line, int nameid, int exec, int visited)
     }
 }
 
+// Mark line as visited, and return true if the line has been previously
+// visited (executed)
+BOOL markVisited(int nameid, UInt line)
+{
+    if (nameid == 0 || line == 0) {
```

Comment:
Can this ever happen, and when?

Reply:
I think I've seen this with SyntaxTree (that doesn't bother setting up this information), and also something the encoder does something weird when special casing some value, for example from memory FALSE and TRUE statements are handled specially. These could get filtered out earlier.

Reply:
Fair enough. Perhaps we should add a brief comment mentioning this?